src/singleton.ts

```
@@ -0,0 +1,45 @@
+import { MeetingParams } from './types'
+
+class Global {
+    private static instance: Global
+    private meetingParams: MeetingParams | null = null
+    public constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)